lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -370,12 +370,12 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
   - a.third.event
           DESCRIPTION
         },
+        # this is only set via server side config
         :apdex_t => {
           :default => 0.5,
-          :public => true,
+          :public => false,
```

Comment:
So it turns out this config is only deprecated for use in the newrelic.yml and env variables. This should only be set via server side config.
The agent however, still uses this value for things so we need to keep the config. So I've removed the deprecated tag and instead have made this a non public config option.